Sources/SecureXPC/Server/XPCServer.swift

```diff
@@ -177,6 +177,36 @@ public class XPCServer {
     // Routes
     private var routes = [XPCRoute : XPCHandler]()
     
+    /// Array of weak references to connections, used to update their dispatch queues
+    internal var connections = [WeakConnection]()
+    
+    /// Weak reference wrapper around an `xpc_connection_t`
+    internal class WeakConnection {
+        weak var connection: xpc_connection_t?
```

Comment:
The idiomatic approach to this is to make a generic `Weak<T>` https://stackoverflow.com/a/24128121/3141234

Reply:
Yeah, I don't think there's any need to monomorphize the generic type. That's the compiler's job.